cmd/log.go

```diff
@@ -4,21 +4,33 @@ import (
 	"fmt"
 	"io"
 	"os"
+	"strings"
 	"time"
 
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/runtime"
+
 	log "github.com/sirupsen/logrus"
 )
 
-func init() {
+// initializeLoggerForConfig will configure the logger so that it matches the
+// user's specified configuration, such as configuring output files.
+func initializeLoggerForConfig(config runtime.Config) error {
 	time := time.Now().Unix()
-	logname := fmt.Sprintf("preflight-%d.log", time)
+	logname := strings.Join([]string{
+		config.CLILogFile, fmt.Sprint(time)}, "-")
```

Comment:
> This this will end up as preflight.log-?

Indeed. Conscious choice. I also tried <timestamp>-<filename> and almost immediately hated that 😆 

> Do we want preflight-.log instead?

This is what we had before, but it's less reliable to do if we accept relatively arbitrary inputs for the filename from the user. I did the above as a result.

> Either way... I want to ask again up front if we can format the time to be '20210707135401' so that at least it is human-readable?
 
Absolutely. I erred on the side of incremental changes and didn't include it in this PR, despite it being effectively a one-line difference. I'll fix it and re-push.

Reply:
Actually, if I'm no longer including a timestamp (and leaving that up to the user), then I think we scratch the need for the human-readable format.